test/uk/gov/hmrc/merchandiseinbaggage/pact/VerifyContractSpec.scala

```diff
@@ -57,7 +56,7 @@ class VerifyContractSpec extends PactVerifySuite with CoreTestData {
           case state: String if state.split("XXX").head == "id1234" =>
             val declarationString = state.split("XXX").toList.drop(1).mkString
             val declaration = Json.parse(declarationString).as[Declaration]
-            repository.insert(declaration).futureValue
```

Comment:
Was .insertDeclaration() not used before? The function was defined before and after, but not .insert() not called, what function was that?

Reply:
so the insert query syntax is changed in HMRC-Mongo’s PlayMongoRepository; defined in DeclarationRepository & used in here.